packages/client/webapp/src/router/index.ts

```diff
@@ -19,4 +26,8 @@ const router = createRouter({
 	routes
 });
 
+export function useIsCurrentRouteHome() {
```

Comment:
This isn't exactly the typical composable pattern, so it might confuse future developers.
- This function depends on router.currentRoute.value.path, which is a reactive dependency changes over time. Just looking at the function signature, though, a developer calling this function might not expect it to change since it's not a computed property or reactive state, and it doesn't have any parameters. I'd recommend either making this a pure function where you pass in `router.currentRoute.value` or make it a computed property directly.
- Composables typically return an object whose properties are all, or almost all, reactive, whereas this function just returns a boolean. If this were a composable its return statement would look something like:
`return { isCurrentRouteHome: computed(() => router.currentRoute.value.path === RoutePath.Home;) };` which seems like overkill to me, a simple computed property would suffice.
- I'm genuinely not sure how using composition API reactivity functions like `computed` would work outside of a component, may need to wrap it in a composable or just move it directly to the App.vue component where it's being used.

Happy to discuss further or have a quick call if you disagree with or are confused by any of that ☝️

Reply:
Hi Ben, 

Good points. You're right, it doesn't really need to be a composable, though I disagree on passing in the value as the caller then needs a reference to router first. It seems simpler and more straight forward to have a function you can call without passing in any parameters. Plus, if you already have the name of the current route then evaluating it is trivial enough to not need separate function anyway.

Reply:
Totally agreed, making it a pure function doesn't really make sense in this case.
My point (buried somewhere in the paragraphs above) is that in general in the app we should have pure functions OR functions with reactive dependencies that live in components or composables.

In the interest of avoiding hard-to-debug reactivity chains as our app grows in size, functions that have reactive dependencies should be distinct. Since we agree pure function isn't the way to go here, I'd recommend creating a composable that exports the `isCurrentRouteHome` computed property.

Then any time this code is used it'll look like
```
const { isCurrentRouteHome } = useEdwinsFancyNewComposable();
const isSidebarVisible = computed(() => isCurrentRouteHome.value === false);
```

and it's impossible to use without that `.value` that signifies it's a reactive dependency that can and will change.

Reply:
@bencodrington Clarification needed: as I understand it so far, `.value` would imply it's a ref. Are you suggesting I wrap `isCurrentRouteHome` in a ref?

Reply:
This has been discussed in a call and resolved.

Thanks for addressing that 👍